docs/wiki/Linux.md

```diff
@@ -0,0 +1,61 @@
+# Linux Guide
+
+## Installation
+
+Go to [Releases](https://github.com/Zagrios/bs-manager/releases) page and go to the latest release. Download the necessary build installer for your distro (see below).
+
+### Universal (flatpak)
+
+You are required to have `flatpak` installed your system. After installing that, download the `.flatpak` file in the releases and run the following command:
+
+```bash
+flatpak install ./bsmanager.flatpak
```

Comment:
Potential change to avoid permission issues:
flatpak install --user ./bsmanager.flatpak